src/components/Navbar/Navbar.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Navbar } from '../index';
+
+describe('Navbar', () => {
+  beforeEach(() => {
+    render(<Navbar />);
+  });
+
+  it('contains header element', () => {
+    const headerElement = screen.getByTestId('header');
```

Comment:
cool, it is a good practice to use data-test-id, to make your tests resilent from changes on structure